tests/test_pointprocess.py

```diff
@@ -0,0 +1,272 @@
+import pytest
+import numpy as np
+
+import parselmouth
+
+
+@pytest.fixture
+def empty_process():
+    return parselmouth.PointProcess()
+
+
+@pytest.fixture
+def poisson_process():
+    return parselmouth.PointProcess.create_poisson_process()
+
+
+@pytest.fixture
+def seqA():
```

Comment:
To be fair, I don't think this needs to be a fixture? I kind of feel like these tests could be a little bit compressed. Mind if I give this a go in a separate commit? We can always revert that if you're not happy with it.

Reply:
My standard disclaimer applies here: do whatever you wish! :-) This was my very first `pytest` I wrote, so it was more of learning process than contribution. So, no issue.